google/cloud/forseti/services/tracing.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Forseti gRPC tracing setup."""
+
+from opencensus.trace.tracer import Tracer
+from opencensus.trace.samplers import always_on
+from opencensus.trace.exporters import stackdriver_exporter, file_exporter
+from opencensus.trace.exporters.transports import background_thread
+from opencensus.trace.ext.grpc import client_interceptor, server_interceptor
+
+from google.cloud.forseti.common.util import logger
+
+LOGGER = logger.get_logger(__name__)
+
+
+def trace_client_interceptor(endpoint):
+    """Intercept gRPC calls on client-side and add tracing information
+    to the request.
+
+    Args:
+        endpoint (str): The gRPC channel endpoint (e.g: localhost:5001).
+
+    Returns:
+        OpenCensusClientInterceptor: a gRPC client-side interceptor.
+    """
+    exporter = setup_exporter()
+    tracer = Tracer(exporter=exporter)
+    return client_interceptor.OpenCensusClientInterceptor(
+        tracer,
+        host_port=endpoint)
+
+
+def trace_server_interceptor():
```

Comment:
Method naming should generally start with verbs.  So, `create_trace_server_interceptor()`.

Reply:
Ok. See previous comment to drop the `_trace`